test/unit/apk-utils-specs.js

```diff
@@ -2,7 +2,8 @@ import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import * as teen_process from 'teen_process';
 import { fs } from '@appium/support';
-import ADB from '../../lib/adb';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
why should it complain about unresolved imports?

Reply:
because without our eslint TS configuration, eslint cannot find a `.ts` file.